client/src/views/DashboardView.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-row class="ma-0 pl-4 pr-4 pt-4 pb-0" align="center">
+    <v-col>
+      <h1>Welcome back, {{ tempUserFirstName }}!</h1>
+      <!-- <v-divider/> -->
+    </v-col>
+  </v-row>
+  <v-divider />
+  <v-row class="ma-0 pl-4 pr-4 pb-0 pt-0" align="center">
+    <v-col>
+      <v-card variant="flat" class="pb-1">
+        <v-container class="pa-0" fluid>
+          <v-row class="ma-0">
+            <v-col cols="auto" class="pl-0 pr-0">
+              <v-icon icon="mdi-run-fast" size="52" />
+            </v-col>
+            <v-col>
+              <v-chip color="green" class="rounded text-h5">{{ tempUserMileage }} KM</v-chip>
+              <v-card-subtitle>TOTAL</v-card-subtitle>
+            </v-col>
+            <v-spacer />
+            <v-col cols="auto">
+              <v-btn
+                variant="elevated"
+                elevated="2"
+                :ripple="true"
+                icon="mdi-plus"
+                color="primaryRed"
+              ></v-btn>
+              <MileageModal v-model="dialog" />
```

Comment:
I cant see the MileageModal component

Reply:
Removed. I can add the MileageModal to the dashboard page in the MileageModal issue